packages/eds-core-react/stories/docs/Intro.stories.mdx

```diff
@@ -1,25 +1,45 @@
 import { Meta } from '@storybook/addon-docs'
-import { Typography, List } from '../../src'
+import { Links, InfoCard, Stack } from '../../.storybook/components/'
+import { Typography, List, Card } from '../../src'
 
 <Meta title="Introduction" />
 
-<Typography variant="h1">Equinor Design System Storybook</Typography>
-<Typography variant="ingress">
-  This is the storybook for the packages of the Equinor Design
-  System. We have a <Typography link href="https://eds.equinor.com/">storefront</Typography> with
-  extensive documentation.
+# Equinor Design System Storybook
+
+<Typography variant="ingress"  style={{marginTop:'24px'}}>
+  This is the storybook for the <Typography link href="https://reactjs.org/">React</Typography> implementation of the Equinor Design
+  System. Our goal is to create and provided building blocks & resources for web development in Equinor.   We have a <Typography link href="https://eds.equinor.com/">storefront</Typography> with
+  extensive documentation for components, guidelines and patterns.
```

Comment:
Should we mention `Figma` with link/links too?

Reply:
Maybe 🤔 I had it initially, but then removed it as we have links to the Figma components on each component. 

I also changed this to be more of a guidelines card. So have another look :)